doc/source/train/distributed-pytorch/using-gpus.rst

```diff
@@ -0,0 +1,149 @@
+Scale, GPUs and other resources
```

Comment:
nit: It's not clear if this should be read as (Scale), (GPUs and other resources), or (Scale), (GPUs) and (other resources).
```suggestion
Configuring scale and resource allocation
```

^ Similar to https://docs.ray.io/en/master/serve/scaling-and-resource-allocation.html

Reply:
I think it's important to have the GPU keyword in the section header. Maybe just `Configuring scale and GPUs`.